float32 mode
float32 x_velocity
float32 y_velocity
float32 z_velocity
float32 yaw_velocity
float32 x_i
float32 y_i
float32 featureset
